src/main.rs

```diff
@@ -82,7 +82,7 @@ fn main() -> Result<()> {
         SubCommand::Kill(kill) => kill.exec(root_path),
         SubCommand::Delete(delete) => delete.exec(root_path, systemd_cgroup),
         SubCommand::State(state) => state.exec(root_path),
-        SubCommand::Info(info) => info.exec(),
+        SubCommand::Info => info::Info::print(),
```

Comment:
We should keep this consistent with all the other commands.

Reply:
Yes, that makes sense :sweat_smile:
I'll close this one and open new one with only comments.